### PR TITLE
[zh-cn] Resync access-authn-authz/authorization.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/access-authn-authz/authorization.md

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -297,6 +297,26 @@ is reachable from the public internet.
...
+privileges, bypassing any authorization restrictions imposed by the RBAC or Webhook mechanisms.
+[Avoid adding users](/docs/concepts/security/rbac-good-practices/#least-privilege)
+to this group. If you do need to grant a user cluster-admin rights, you can create a
+[ClusterRoleBinding](/docs/reference/access-authn-authz/rbac/#user-facing-roles)
+to the built-in `cluster-admin` ClusterRole.
+-->
+### `system:masters` 组
+
+`system:masters` 组是 Kubernetes 内置的一个组，授予其成员对 API 服务器的无限制访问权限。
+任何被分配到此组的用户都具有完整的集群管理员权限，可以绕过由 RBAC 或 Webhook 机制施加的任何授权限制。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:314]:
```suggestion
任何被分配到此组的用户都具有完全的集群管理员权限，可以绕过由 RBAC 或 Webhook 机制施加的任何鉴权限制。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 2]
```diff
@@ -297,6 +297,26 @@ is reachable from the public internet.
...
+[避免将用户添加到此组](/zh-cn/docs/concepts/security/rbac-good-practices/#least-privilege)。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:315]:
```suggestion
请[避免将用户添加到此组](/zh-cn/docs/concepts/security/rbac-good-practices/#least-privilege)。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 3]
```diff
@@ -297,6 +297,26 @@ is reachable from the public internet.
...
+如果你确实需要授予某个用户集群管理员权限，可以通过创建一个
+[ClusterRoleBinding](/docs/reference/access-authn-authz/rbac/#user-facing-roles)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:317]:
```suggestion
[ClusterRoleBinding](/zh-cn/docs/reference/access-authn-authz/rbac/#user-facing-roles)
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**